mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/ReportsColumns.tsx

```diff
@@ -745,47 +745,53 @@ export const Columns = (props: ColumnsProps) => {
 
         return (
           <VerticalContainer className={styles.list}>
-            {row.original.comments
-              .map((comment, index) => (
-                <VerticalContainer
-                  key={comment.uuid}
-                  className={styles.verticalContainer}
-                >
-                  <HorizontalContainer className={styles.horizontalContainer}>
-                    <HorizontalContainer className={styles.listNumberAndName}>
-                      {getListNumberByIndex(index)}
-                      <Link path={pages.user.getPath({uuid: comment.ownerUuid})}>
-                        {getName(props.way, comment.ownerUuid)}
-                      </Link>
-                    </HorizontalContainer>
-                    {comment.ownerUuid === user?.uuid &&
-                      <Tooltip
-                        content="Delete comment"
-                        position={PositionTooltip.LEFT}
-                      >
-                        <Confirm
-                          trigger={<TrashIcon className={styles.icon} />}
-                          content={<p>
-                            {`Are you sure you want to delete the comment "${comment.description}"?`}
-                          </p>}
-                          onOk={() => deleteComment(comment.uuid)}
-                          okText="Delete"
+            <ol className={styles.numberedList}>
```

Comment:
why do we need ol here?

Reply:
i see for now we have deeper nesting, could we get rid of some container

Reply:
Done